python/pyarrow/_compute.pyx

```diff
@@ -1458,10 +1459,12 @@ class StrptimeOptions(_StrptimeOptions):
     unit : str
         Timestamp unit of the output.
         Accepted values are "s", "ms", "us", "ns".
+    error_is_null : boolean, default False
+        Return null on parsing errors if true or raise if false.
     """
 
-    def __init__(self, format, unit):
-        self._set_options(format, unit)
+    def __init__(self, format, unit, error_is_null=True):
```

Comment:
This should be False for the default? 

Can you also add a test for this default? (a test that checks an error is raised for the default options without explicitly passing `error_is_null`)

Reply:
Ah sorry, it was a remainder from the previous `raise_errors` parameter. Changed and added a test.